src/form-workflows.rst

```diff
@@ -0,0 +1,215 @@
+
+When using one submission per observation, :ref:`last-saved <last-saved>` can be used to reduce how much duplicated information is entered.
+
+* :fa:`external-link` `Example of a form that uses last-saved to reduce entry of common data <https://docs.google.com/spreadsheets/d/1fh6lnFPAJ8FWggqadvQkq_x2QKH2pBVEkXlEFr8h_OI>`_
+* :fa:`external-link` `Example of a form that uses repeats to reduce entry of common data <https://docs.google.com/spreadsheets/d/10wu2ezTYNg_zuNOcKB-pbRdc4DamWrT7ooNN_vp5nh8>`_
+
+Advantages of repeats
+~~~~~~~~~~~~~~~~~~~~~~~
+
+* Less need to navigate around Collect: users swipe forward in a form and are prompted to capture a new observation.
+* Logic can be used between repeat instances (this can be done with either an external app (e.g. `counter <https://github.com/getodk/counter>`_) or :ref:`last-saved <last-saved>` but neither supports submission edits). For example:
+
+  * Keep capturing plants until you have observed at least 10 of the same species
+  * Sequentially number households
+
+Disadvantages of repeats
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* Form can get slow or hard to navigate as the number of repeat instances increases
+* Data can’t be downloaded or viewed until all observations are captured and the full form is sent
+* The repeats and the top-level data will generally need to be joined in analysis (see the tip in :ref:`the repeats section <repeats>`)
+
+Location-centric workflows
+---------------------------
+Many workflows have a geo-spatial component to them. For example, a conservation workflow may involve periodically visiting and assessing specific old-growth trees.
+
+The form :doc:`submission map <collect-form-map>` for a form can be used as the landing screen for location-centric workflows. It can help plan travel and display completed work.
+
+Other tools for location-centric workflows include :ref:`select one from map <select-from-map>` and :doc:`offline layers <collect-offline-maps>`. External apps can also be used for things like wayfinding and navigation [link].
+
+Using Collect settings for workflow support 
+--------------------------------------------
+ODK Collect is :doc:`highly configurable <collect-settings>` and different combinations of options will better support some workflows than others. Once you design an ideal workflow for the tasks you need to accomplish, we recommend looking at the :doc:`available settings <collect-settings>` and considering which could help support your workflow. Consider what you want the user experience to be before entering your form, during form filling and after completing the tasks represented by the form. If you change default settings, we recommend you configure one device as desired and :ref:`create a QR code to configure others <sharing-settings-with-another-device>`.
+
+Below are a few examples of settings that can help support common workflows. 
+
+Before form entry
```

Comment:
These sections could also be thought of as "form availability|navigation|submission" 

Or maybe...."Which form to fill and when, how to navigate a form, sending and keeping finished forms